jansi/src/main/java/org/fusesource/jansi/AnsiConsole.java

```diff
@@ -129,7 +129,7 @@ public void close() throws IOException {
         };
     }
 
-    private static boolean isCygwin() {
+    private static boolean isXterm() {
```

Comment:
this one is not about example
but it seems a good change: yes, given the implementation, this method should be renamed!
+1